src/launch/mod.rs

```diff
@@ -348,39 +348,25 @@ bitflags! {
 /// Encoded page types for a launch update. See Table 58 of the SNP Firmware
 /// specification for further details.
 #[derive(Copy, Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
+#[repr(C)]
```

Comment:
Also: `#[non_exhaustive]`.

Basically, all `enum`s should have `#[non_exhaustive]` unless there is very strong reason to avoid it.

Reply:
right